store/src/java/com/zimbra/cs/filter/jsieve/ReplaceHeader.java

```diff
@@ -125,6 +125,9 @@ protected Object executeBasic(MailAdapter mail, Arguments arguments,
                                     newHeaderValue = header.getValue();
                                 }
                                 ZimbraLog.filter.debug("replaceheader: header after processing\n%s: %s", newHeaderName, newHeaderValue);
```

Comment:
can remove this logging at debug level as same info is now logged at info level.